### ISSUE TITLE
cmake target name

### ISSUE THREAD

**Issue by loriab** [wavefunction91/IntegratorXX#77]
I think y

**Comment by wavefunction91** [comment]
Yes, but I alias to the `IntegratorXX::IntegratorXX` target here
https://github.com/wavefunction91/IntegratorXX/blob/master/CMakeLists.txt#L8

That should allow you to link downstream to the same target either source included or discovered. We do this in GauXC here
https://github.com/wavefunction91/GauXC/blob/master/src/CMakeLists.txt#L75-L77

Does the location property deduction fail? I.e. is this alias causing build symptoms?

**Comment by loriab** [comment]
Thanks for the fix!
